### PR TITLE
Localized 10 Additional Slides

### PR DESCRIPTION
**Pull request by rcoop017**
Localized 10 additional slides.

See Commits, there were a few that I couldn't localize entirely due to weird && statements...

### CODE REVIEW

File: src/components/application/slides/EmploymentIncome.jsx [hunk 1]
```diff
@@ -22,24 +23,62 @@ class EmploymentIncome extends Component {
...
+          <FormattedMessage
+              id="app.slides.employmentIncome.tips"
+              description="Tips"
+              defaultMessage="Tips"
+          />
         </IncomeSource>
 
-        <IncomeSource incomeSources={incomeSources} name="commission">
-          Commission
+        <IncomeSource incomeSources={incomeSources} name="Commission">
```
Comment (ambit-developer) [inline src/components/application/slides/EmploymentIncome.jsx]: I think the `name` attribute here needs to stay lower case, as it's a key into an object property.

File: src/components/application/slides/EmploymentIncome.jsx [hunk 2]
```diff
@@ -22,24 +23,62 @@ class EmploymentIncome extends Component {
...
+          <FormattedMessage
+              id="app.slides.employmentIncome.Commission"
+              description="Commission"
+              defaultMessage="Commission"
+          />
         </IncomeSource>
 
         <IncomeSource incomeSources={incomeSources} showAnnual={true} name="cashBonus">
```
Comment (ambit-developer) [inline src/components/application/slides/EmploymentIncome.jsx]: The below chunk could be unwrapped — don't need an outer `FormattedMessage` that's just a tooltip.

Also, the unlocalized tooltip still exists.

File: src/components/application/slides/IncomeTypeDefaultText.jsx [hunk 3]
```diff
@@ -12,15 +13,41 @@ class IncomeTypeDefaultText extends Component {
...
-          Does <strong>{informalName(person)}</strong> have income from the following sources{!!showMilitaryCaveat && ', not including earnings from the military that were already reported'}?
+          <FormattedMessage
+              id="app.slides.IncomeTypeDefaultText.intro"
+              description="Introductory paragraph."
+              defaultMessage="Does {person} have income from the following sources "
+              values={{
+                person: <strong>{informalName(person)}</strong>
+              }}
+          />
+          {!!showMilitaryCaveat && ', not including earnings from the military that were already reported'}?
```
Comment (ambit-developer) [inline src/components/application/slides/IncomeTypeDefaultText.jsx]: The caveat string can be localized in place, e.g.

```JSX
{!!showMilitaryCaveat &&
 <FormattedMessage … />
}
```

**Comment by rcoop017** [comment]
Check out the changes, I think everything's good to go now.

**Pull request merged**